vue/src/views/Show.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="show">
+    <Header />
+    <h1>User Name</h1>
+    <div class="Chart">
```

Comment:
```suggestion
    <div class="chart">
```
class名はキャメルケース、もしくはケバブとかで会わしてください。